packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2398,35 +2404,26 @@ class _StandardBottomSheetState extends State<_StandardBottomSheet> {
 
   @override
   Widget build(BuildContext context) {
-    if (widget.animationController != null) {
-      return AnimatedBuilder(
-        animation: widget.animationController,
-        builder: (BuildContext context, Widget child) {
-          return Align(
-            alignment: AlignmentDirectional.topStart,
-            heightFactor: widget.animationController.value,
-            child: child
-          );
-        },
-        child: _wrapBottomSheet(
-          BottomSheet(
-            animationController: widget.animationController,
-            enableDrag: widget.enableDrag,
-            onClosing: widget.onClosing,
-            builder: widget.builder,
-            backgroundColor: widget.backgroundColor,
-            elevation: widget.elevation,
-            shape: widget.shape,
-          ),
+    return AnimatedBuilder(
```

Comment:
This change doesn't seem to be related to the PR's description. If animationController is guaranteed to be non-null, are we missing some asserts? Maybe this change should be made in a separate PR?

Reply:
I'll undo this change as it isn't related, you're right